team-telnyx/webrtc
# Add React media components

Adds `Media`, `Audio` and `Video` components, with examples, to `react-client`.

## 📝 To Do


## ✋ Manual testing

Note: I'm testing with the [Telnyx web dialer](https://webrtc.telnyx.com/rtc/index.html), but should work in theory with a hardware or softphone, too.
1. Build react-client by running `npm run build` in `packages/react-client`
2. `cd` into `packages/react-client/example` and run `npm start`
3. Make an audio-only call. Verify that you hear audio coming from the browser
4. Make a video call. Verify that you see and hear video in the browser

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

<img width="2079" alt="Screen Shot 2020-11-12 at 12 10 20 PM" src="https://user-images.githubusercontent.com/4672952/98991483-be5d6c80-24e0-11eb-892e-fa36857ac602.png">
